src/ThreeEditor/Simulation/Scoring/ScoringManager.ts

```diff
@@ -110,7 +117,9 @@ export class ScoringManager
 	getFilterOptions(): Record<string, string> {
 		const options = this.filters
 			.filter(filter => {
-				return filter.rules.length;
+				if (filter instanceof CustomFilter) return filter.rules.length;
+
+				return true;
```

Comment:
Simplified using ternary operator.

```suggestion
			.filter(filter => filter instanceof CustomFilter ? filter.rules.length : true)
```